code-review/README.md

```diff
@@ -31,7 +31,8 @@ Having Your Code Reviewed
 
 * Be grateful for the reviewer's suggestions. ("Good call. I'll make that
   change.")
-* Don't take it personally. The review is of the code, not you.
+* Be aware of [how hard it is to convey emotion online] and how easy it is to
+  misinterpret feedback.
```

Comment:
> A common axiom is "Don't take it personally. The review is of the code, not you."
> We used to include this, but now prefer to say what we mean:
Be aware of [how hard it is to convey emotion online] and how easy it is to misinterpret feedback.

I really like this. While I understand that people may not be able to divorce themselves from their code, we have a culture of shared success, achievement and failure. Once your code is merged, it's part of the team so reviewing the code and not the person, is the goal.

Maybe add something like :
> Assume the best intention from the reviewer's comments.